signinapp/admin/users.py

```diff
@@ -77,15 +93,14 @@ def new_user():
 @admin.route("/admin/users/edit", methods=["GET", "POST"])
 @admin_required
 def edit_user():
-    user : User = User.query.get(request.args["user_id"])
+    user: User = User.query.get(request.args["user_id"])
```

Comment:
Please use `db.session.get(User, request.args["user_id"])` for this - see [here](https://docs.sqlalchemy.org/en/14/changelog/migration_20.html#orm-query-get-method-moves-to-session) where the query object is deprecated